src/task_manager/sensor_checker_state.py

```diff
@@ -0,0 +1,70 @@
+import rospy
+
+from geometry_msgs.msg import Accel
+from sensor_msgs.msg import Imu, MagneticField, NavSatFix, FluidPressure
+from nav_msgs.msg import Odometry
+from gazebo_msgs.msg import ModelStates
+from rosgraph_msgs.msg import Clock
+
+
+
+class Sensor_Checks(smach.State):
+    def __init__(self, time_out):
+        smach.State.__init__(self, outcomes=['Checks_passed', 'Checks_failed'])
+        # rospy.init_node('sensor checker', anonymous=True)
+
+        self.Imu_check = False
```

Comment:
The naming is weird here. If you use underscores, everything is lower case. If you use capitalization, only capitalize the first letter of each word and don't use underscores